src/ResourceManager/DeploymentManager/Commands.DeploymentManager/Commands/GetRollout.cs

```diff
@@ -88,6 +94,12 @@ public override void ExecuteCmdlet()
             }
 
             var rollout = this.DeploymentManagerClient.GetRollout(this.ResourceGroupName, this.Name, this.RetryAttempt);
+
+            if (this.ShowDetails)
+            {
+                RolloutCmdletBase.PrintRollout(rollout);
```

Comment:
@deveshguha I think the best way to model this is to change the `Console.WriteLine(...)` calls to the appropriate stream writing calls, and then get rid of this new `-ShowDetails` parameter and _always_ make the calls to write to the verbose and error streams. This means that if the user ever wants the extra information that you're reporting back, they will always get it if they provide the `-Verbose` or `-Error` switch parameter that automatically is included with every cmdlet that we shipped (and is handled by the PowerShell engine). Let me know if you have any comments or concerns about this.